src/PKSim.Core/Model/IndividualTransporterFactory.cs

```diff
@@ -47,6 +42,18 @@ public IndividualTransporter UndefinedLiverTransporterFor(Individual individual)
          return transporter;
       }
 
+      public IndividualMolecule CreateFor(ISimulationSubject simulationSubject, TransportType transporterType)
```

Comment:
Another bug that I found trying to compare old and new. old batch always created an efflux, no matter what. Here we will create the real process type